pkg/sdk/helper_test.go

```diff
@@ -46,3 +48,10 @@ func testClient(t *testing.T, profile string) *Client {
 
 	return client
 }
+
+func configAccountId(t *testing.T, cfg *gosnowflake.Config) AccountIdentifier {
```

Comment:
this is repeated but in my service user on CI change there will be other alternative instead of this func, so nothing to do yet